// stripe/esm/resources/Identity/VerificationSessions.js
[Readme reference] This project, Restaurant Menu, is part of the Aenzbi suite. For more details, refer to the README.md file.

/*
 * aenzbi - Business Management Software
 * 
 * Copyright (c) 2024 Ally Elvis Nzeyimana
 * 
 * This software is licensed under the MIT License. See the LICENSE file for details.
 */

// File generated from our OpenAPI spec
import { StripeResource } from '../../StripeResource.js';
const stripeMethod = StripeResource.method;
export const VerificationSessions = StripeResource.extend({
    create: stripeMethod({
        method: 'POST',
        fullPath: '/v1/identity/verification_sessions',
    }),
    retrieve: stripeMethod({
        method: 'GET',
        fullPath: '/v1/identity/verification_sessions/{session}',
    }),
    update: stripeMethod({
        method: 'POST',
        fullPath: '/v1/identity/verification_sessions/{session}',
    }),
    list: stripeMethod({
        method: 'GET',
        fullPath: '/v1/identity/verification_sessions',
        methodType: 'list',
    }),
    cancel: stripeMethod({
        method: 'POST',
        fullPath: '/v1/identity/verification_sessions/{session}/cancel',
    }),
    redact: stripeMethod({
        method: 'POST',
        fullPath: '/v1/identity/verification_sessions/{session}/redact',
    }),
});
